src/plib/controllers/IndexController.php

```diff
@@ -133,7 +133,7 @@ private function getFolderList($path)
         ]);
 
         $list->setData($data);
-        $list->setDataUrl(['action' => 'index-data']);
+        $list->setDataUrl(['action' => 'index-data?path=' . $path]);
```

Comment:
let's create a full url with view helper, because parameters inside "action" look quite dirty.
`$list->setDataUrl($this->view->url(['action' => 'index-data', 'path'  => $path]))`